lib/parse.js

```diff
@@ -35,10 +35,12 @@ Falling back to API Blueprint by default.
     let nativeError = null;
 
     if (!(err || apiElements)) {
-      nativeError = new Error('Unexpected parser error occurred.');
+      nativeError = new Error('Unexpected parser error occurred');
     } else if (err) {
       // Turning Fury error object into standard JavaScript error
       nativeError = new Error(err.message);
+    } else if (apiElements && apiElements.errors.length) {
```

Comment:
```suggestion
    } else if (apiElements && !apiElements.errors.isEmpty) {
```

Using `isEmpty` may be clearer to express the intent.